Tests/UnitTests/CMakeLists.txt

```diff
@@ -5,6 +5,11 @@ endif()
 set(SCRIPTS
     "Scripts/tests.js")
 
+if(APPLE OR UNIX OR WIN32)
```

Comment:
Android is Unix. so, if I'm not wrong this can be simplified to (NOT WINDOW_STORE). Or is Android expected to not run this code block?

Reply:
I was not able to create the symlinks needed for testing on Android (and iOS), so I left those platforms out of the unit tests.